tm2/pkg/crypto/keys/types.go

```diff
@@ -44,7 +44,8 @@ type Keybase interface {
 	// CreateMulti creates, stores, and returns a new multsig (offline) key reference
 	CreateMulti(name string, pubkey crypto.PubKey) (info Info, err error)
 
-	// Rotate replaces the encryption password for a given key
+	// Rotate replaces the encryption password for a given key.
+	// In all import operations, if an account exists with the same address but a different name, it is replaced by the new name
```

Comment:
This comment is still true, but it only applies to `ImportPrivKey` (not to `Rotate`).

Reply:
Updated:

[21416bd](https://github.com/gnolang/gno/pull/3285/commits/21416bd8687f51fc24806e867e7a51bcde5e922f)